packages/cli/README.md

```diff
@@ -334,9 +347,175 @@ ARGUMENTS
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:add STREAMID`
+
+pin a stream
+
+```
+USAGE
+  $ glaze pin:add STREAMID
+
+ARGUMENTS
+  STREAMID  ID of stream to be pinned
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:ls [STREAMID]`
+
+list pinned streams
+
+```
+USAGE
+  $ glaze pin:ls [STREAMID]
+
+ARGUMENTS
+  STREAMID  optional stream ID filter
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:rm STREAMID`
+
+unpin a stream
+
+```
+USAGE
+  $ glaze pin:rm STREAMID
+
+ARGUMENTS
+  STREAMID  ID of stream to be unpinned
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze stream:commits STREAMID`
+
+list commits contained within a stream
+
+```
+USAGE
+  $ glaze stream:commits STREAMID
+
+ARGUMENTS
+  STREAMID  ID of the stream
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
```

Comment:
okay gonna stop writing ditto, but what does --key do for any read-only operation?  I would assume it would only do anything for commands that result in new commits being authored